curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61b0a44a6b865738ba49b9fb.md

```diff
@@ -15,6 +15,15 @@ The `box-shadow` property lets you apply one or more shadows around an element.
 box-shadow: offsetX offsetY color;
 ```
 
+Here's how the `offsetX` and `offsetY` values work:
+
+* both `offsetX` and `offsetY` accepts number values in `px` and other CSS units
+* a positive `offsetX` value moves the shadow right and a negative value moves it left
+* a positive `offsetY` value moves the shadow down and a negative value moves it up
+* if you want a value of zero (`0`) for any or both `offsetX` and `offsetY`, you don't need to add a unit. Every browser understands that zero means nothing. This does not apply to box shadows only, its a CSS thing.
```

Comment:
"This does not apply to box shadows only, its a CSS thing." I would remove this last sentence, as it may confuse people.

Reply:
I have removed the line. I hope it's not too verbose too. I did what the feedback stated. Same with the case-sensitive one for `A-F` too.

Reply:
OK thanks for doing this. I think this will reduce the total text without losing too much information :)